l10n.js

```diff
@@ -1,74 +1,17 @@
-    "curators":"Los curadores del EDS son"
+    "language": "English",
+    "translator": "",
+    "unique": "Celebrate the 100th SDS with us by creating a project with 10 squares!\n<br/><br/>  Create a project that contains 10 squares on the screen at a time, and 10 squares only. It could be a train driving on a track, with the train made out of 7 squares, and the tracks made out of 3. It could be an Ice-cream simulator with the ice cream made out of 5 squares, and the ice cream machine made out of the other 5 squares. The possibilities are endless!<br/><br/> Wondering why we're making things out of 10 squares? It's because in math, 10 squared equals 100, and squared sounds like squares!<br/>",
+    "standard": "o have your project added to this studio, please post a comment with a link. You can find a link to your project by clicking the ‘Embed’ button on the project page. The link will be directly below the Facebook logo.\n<br/>Please submit a single project made no earlier than one month before the start of this Design studio. However, you can update your project, and it'll be eligible again. If you have an older project that would fit, add some new features and feel free to submit the updated version!\n<br/>All projects must meet the community guidelines and be appropriate for all ages!\n<br/>It may take some time for your projects to be added. Please be patient, we add projects oldest to newest; no need to double post!\n<br/><br/>SDS (Scratch Design Studio) is a themed studio where any Scratcher can submit a project that matches the current theme so long as it meets the requirements shown above.\n Check out this project to see how new SDS curators are selected: <a href=\"https://scratch.mit.edu/projects/55738732/\">https://scratch.mit.edu/projects/55738732/</a>\n Please do not simply ask to be a curator!\n<br/><br/>Do you have an idea for a future design studio? Visit the studio linked below, read the studio description, and then submit your idea in the form of a project here: <a href=\"https://scratch.mit.edu/studios/93627/\">https://scratch.mit.edu/studios/93627/</a>\n<br/><br/>Questions about Scratch Design Studio? Check out the Scratch Wiki's article on it: <a href=\"http://wiki.scratch.mit.edu/wiki/SDS/\">http://wiki.scratch.mit.edu/wiki/SDS/</a>\n<br/><br/>Check out this account for new updates on the SDS, new SDS curators and more! <a href=\"https://scratch.mit.edu/users/SDS-Updates/\">@SDS-Updates</a>\n<br/><br/>Credit to @SDScurator1 and @SDScurator2 for the thumbnail!",
+    "curators": "The SDS is curated by"
   },
   "fr": {
```

Comment:
We'd prefer to keep hand translated translations over Google translated translations.